chromium_src/third_party/blink/public/mojom/worker/worker_content_settings_proxy.mojom

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// This file contains patches to apply to an original mojom file.
+module blink.mojom;
+
+[BraveExtend]
+interface WorkerContentSettingsProxy {
+  // Returns whether the worker is allowed access to privileged functions that could be used for fingerprinting.
+  [Sync]
+  AllowFingerprinting() => (bool result);
+  [Sync]
+  GetBraveFarblingLevel() => (uint8 result);
```

Comment:
It's possible to use `enum` here instead of `uint8`, but you have to create `EnumTraits` and register it in `BUILD.gn`. You might want to try it, but I'm also fine with the current impl.

`EnumTraits` example: [c++ part](https://github.com/chromium/chromium/blob/98.0.4750.1/components/content_settings/core/common/content_settings_mojom_traits.cc#L32-L81),  [BUILD.gn part](https://github.com/chromium/chromium/blob/98.0.4750.1/components/content_settings/core/common/BUILD.gn#L70-L103).

Reply:
I am also fine with the current implementation.